source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,62 @@
+.. include:: <isonum.txt>
+
+Robot Battery
+=============
+
+The power supply for an FRC\ |reg| robot is a single 12V 18Ah battery. The batteries used for FRC are sealed lead acid batteries capable of meeting the high current demands of an FRC robot.
+
+Battery Connections
+^^^^^^^^^^^^^^^^^^^
+
+Great care should be taken when handling batteries.  **DO NOT CARRY A BATTERY BY THE CABLES**.  The battery will not be able to provide the same amount of current if the :ref:`connectors are loose <docs/hardware/hardware-basics/preemptive-troubleshooting:Check Battery Connections>`.
+
+.. image:: images/robot-battery/heatshrink.png
+   :alt: One terminal of an FRC battery fully covered in heatshrink.
+
+It is a good idea to secure the battery with either a star washer or a nord lock washer as this will ensure the connection does not move.  This connection must also be wrapped, while some use electrical tape heatshrink that fits over the entire connection is recommended.
+
+Battery Longevity
+^^^^^^^^^^^^^^^^^
+
+This battery is rated for about 1200 charge/recharge cycles, but for the high currents FRC subjects it to that lifespan is reduced to about 400 cycles.  These cycles are intended to be relatively limited discharge and deep cycling the battery (running it all the way down) can damage it.
```

Comment:
> This battery is rated for about...

I'm not a huge fan of "this", rather would replace it with a name or noun.

Are all FRC batteries equal? Are they all rated for 1200 charge/recharge?

Reply:
they are not all equal but that is why it was qualified with about.  The point I am trying to make here is that no matter what the published cycle rating is general FRC use and a few other factors significant reduce the amount of cycles the battery can take.